Gem/Swap/ViewModels/SwapViewModel.swift

```diff
@@ -47,6 +47,8 @@ class SwapViewModel {
     var fromValue: String = ""
     var toValue: String = ""
 
+    var fetchState: SwapFetchState = .idle
```

Comment:
var fetchState: SwapFetchState = .idle
    var swapAvailabilityState: StateViewType<SwapAvailabilityResult> = .noData
    var swapGetQuoteDataState: StateViewType<Bool> = .noData

too many states, can we consolidate it?